Documentation/kubernetes.md

```diff
@@ -37,6 +37,71 @@ kubectl create -f rook-operator.yaml
 kubectl -n rook-system get pod
 ```
 
+ 
+#### RBAC for PodSecurityPolicies
+
+If you have activated the [PodSecurityPolicy Admission Controller](https://kubernetes.io/docs/admin/admission-controllers/#podsecuritypolicy) and thus are
+using [PodSecurityPolicies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/), you will require additional 
+(Cluster)RoleBindings for the different ServiceAccounts Rook uses to start the Rook Agent, CEPH Mons and OSD Pods.
```

Comment:
Could you please put `ClusterRoleBindings` and `ServiceAccount`s in a code block.

Also it isn't only the Ceph Mons and OSD Pods, writing overall the "rook storage Pods" for example is better for the future features.